LEGO1/lego/legoomni/src/worlds/historybook.cpp

```diff
@@ -141,10 +141,7 @@ void HistoryBook::ReadyWorld()
 #ifdef BETA10
 						memset(m_scores[i]->GetBitmapStart(scoreboxX, scoreboxY + lax), scoreColors[color - 1], 4);
 #else
-						if (m_scores[i]->GetAlphaMask() != NULL) {
-							memset(NULL, scoreColors[color - 1], 4);
```

Comment:
I'd just comment out the memset call, but leave the branch intact. The more code we can keep unchanged the better